content/04.results.md

```diff
@@ -3,11 +3,62 @@
 ## Random Sampling of Generative Model
 place the grid aurocs here for generative model
 
-## Discriminator Model Builds Off Generative Model
-place the grid of aurocs here for discriminator model
+### Discriminator Model Builds Off Generative Model
+We evaluate the discriminator model's ability to build off of the generative model.
+Each generative model, trained on randomly sampled label functions, generates probabilistic training labels for each sentence in our dataset.
+For example, one sentence could receive a high confidence score (>=0.6), a low score (0.49<=) or it could receive a borderline score of 0.5.
+Using these confidence scores, the discriminator model constructs sentence features and generates a final confidence score (0-1) for each sentence in our dataset.
+We report the results of the discriminative model using area under the precision recall curve (AUPRC) (Figure {@fig:discriminative_model_auprc}) and area under the receiver operating curve (AUROC) (Figure {@fig:discriminative_model_auroc}).    
+Overall the discriminative model performs well when using a generative model that is trained off of edge specific label functions. 
```

Comment:
Wait - isn't the disciminative model usually worse than the generative model? Am I misreading the results.